packages/common/src/utils/updatePlaylistArtwork.ts

```diff
@@ -76,23 +80,34 @@ export const updatePlaylistArtwork = async (
       tracksForImage = actions.updated.slice(0, 4)
     }
   } else if (actions.regenerate) {
-    tracksForImage = tracks.slice(0, 4)
+    if (tracks.length < 4) {
+      tracksForImage = tracks.slice(0, 1)
+    } else {
+      tracksForImage = tracks.slice(0, 4)
+    }
   }
 
   if (tracksForImage) {
-    const trackUrls = await Promise.all(
-      tracksForImage.map(async (track) => {
-        const { cover_art_sizes, cover_art } = track
-        return await context.audiusBackend.getImageUrl(
-          cover_art_sizes ?? cover_art,
-          SquareSizes.SIZE_1000_BY_1000
-        )
-      })
-    )
+    if (tracksForImage.length === 0) {
+      console.log('okay full removal!')
+      // @ts-expect-error
```

Comment:
still expecting this error?

Reply:
yes actually, apparently it cant be nullable?